src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -491,6 +492,19 @@ public IContractResolver JsonContractResolver
                 _findCollectionNameForDynamic = value;
             }
         }
+        
+        /// <summary>
+        ///     Gets or sets the function to find the collection name for dynamic type.
+        /// </summary>
+        public Func<dynamic, string> FindTypeNameForDynamic
```

Comment:
Shouldn't this be `FindClrTypeNameForDynamic` to match the other convention `FindClrTypeName` ?